### PR TITLE
Feature/saml auth provider

### PR DESCRIPTION
**Pull request by kevintechie**
adding the ability to use the passport-saml authentication provider with loopback

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "security"
   ],
   "homepage": "https://github.com/strongloop/loopback-component-passport",
-  "version": "1.5.0",
+  "version": "1.6.0",
```
Comment (raymondfeng) [inline package.json]: Please do not change the version as we have a release tool to automate that.

**Comment by raymondfeng** [comment]
@kevintechie Thank you for the patch. Your commit has both formatting and functional changes. It makes the review quite difficult. Would you please revert the formatting changes (or submitting such changes in a separate PR)?

**Comment by kevintechie** [comment]
I reverted and committed the saml patch.

The lint and formatting changes were easier to see in my diff tool, but GitHub kind of made a mess of it.


(K)

---- On Thu, 10 Sep 2015 14:24:38 -0700 Raymond Feng &lt;notifications@github.com&gt; wrote ---- 

@kevintechie Thank you for the patch. Your commit has both formatting and functional changes. It makes the review quite difficult. Would you please revert the formatting changes (or submitting such changes in a separate PR)?

**Pull request closed without merging**